pkgs/development/python-modules/django-health-check/default.nix

```diff
@@ -0,0 +1,45 @@
+{ lib
+, fetchPypi
+, buildPythonPackage
+, pytest-runner
+, sphinx
+, setuptools_scm
+, django
+, redis
+, celery
+, pytest-django
+, pytest-cov
+, pytest
+, mock }:
+
+buildPythonPackage rec {
+  pname = "django-health-check";
+  version = "3.16.5";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "sha256-Ht/UkpPM67zin52mCcQH8weu4kCreZq0IBAxNBrnjA8=";
+  };
+
+  propagatedBuildInputs = [
+    pytest-runner
+    sphinx
+    setuptools_scm
+    django
+    redis
+    celery
+    pytest-django
+    pytest-cov
+    pytest
+    mock
+  ];
+  # Disable check because it requires a running Django instance
+  doCheck = false;
```

Comment:
That is easily possible and upstream CI does nothing out of the ordinary https://github.com/KristianOellegaard/django-health-check/blob/master/.github/workflows/ci.yml. Most likely adding pytest-django to checkInputs will already fix it.

Reply:
The pypi tarball is missing the tests folder so this should be the reason for the failing tests. If I fetch it from GitHub I get a different error: 

```
Executing setuptoolsBuildPhase
Traceback (most recent call last):
  File "/build/source/nix_run_setup", line 8, in <module>
    exec(compile(getattr(tokenize, 'open', open)(__file__).read().replace('\\r\\n', '\\n'), __file__, 'exec'))
  File "setup.py", line 4, in <module>
    setup(use_scm_version=True)
  File "/nix/store/nf0agngli8a8ffqqcx60vswsmg1khran-python3.9-setuptools-61.2.0/lib/python3.9/site-packages/setuptools/__init__.py", line 87, in setup
    return distutils.core.setup(**attrs)
  File "/nix/store/nf0agngli8a8ffqqcx60vswsmg1khran-python3.9-setuptools-61.2.0/lib/python3.9/site-packages/setuptools/_distutils/core.py", line 109, >
    _setup_distribution = dist = klass(attrs)
  File "/nix/store/nf0agngli8a8ffqqcx60vswsmg1khran-python3.9-setuptools-61.2.0/lib/python3.9/site-packages/setuptools/dist.py", line 462, in __init__
    _Distribution.__init__(
  File "/nix/store/nf0agngli8a8ffqqcx60vswsmg1khran-python3.9-setuptools-61.2.0/lib/python3.9/site-packages/setuptools/_distutils/dist.py", line 293, >
    self.finalize_options()
  File "/nix/store/nf0agngli8a8ffqqcx60vswsmg1khran-python3.9-setuptools-61.2.0/lib/python3.9/site-packages/setuptools/dist.py", line 886, in finalize>
    ep(self)
  File "/nix/store/nf0agngli8a8ffqqcx60vswsmg1khran-python3.9-setuptools-61.2.0/lib/python3.9/site-packages/setuptools/dist.py", line 907, in _finaliz>
    ep.load()(self, ep.name, value)
  File "/nix/store/wvih3p12zc131j812c414d2mfff79m4v-python3.9-setuptools-scm-6.4.2/lib/python3.9/site-packages/setuptools_scm/integration.py", line 75>
    _assign_version(dist, config)
  File "/nix/store/wvih3p12zc131j812c414d2mfff79m4v-python3.9-setuptools-scm-6.4.2/lib/python3.9/site-packages/setuptools_scm/integration.py", line 51>
    _version_missing(config)
  File "/nix/store/wvih3p12zc131j812c414d2mfff79m4v-python3.9-setuptools-scm-6.4.2/lib/python3.9/site-packages/setuptools_scm/__init__.py", line 106, >
    raise LookupError(
LookupError: setuptools-scm was unable to detect version for /build/source.

Make sure you're either building from a fully intact git repository or PyPI tarballs. Most other sources (such as GitHub's tarballs, a git checkout wi>

For example, if you're using pip, instead of https://github.com/user/proj/archive/master.zip use git+https://github.com/user/proj.git#egg=proj
```